File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/annotation/HContentViewAttachToRoot.java
package com.hw.hwdroid.foundation.app.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by ChenJ on 16/8/17.
 */
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface HContentViewAttachToRoot {

    boolean value() default false;

}

File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/annotation/HContentViewAttachToRoot.kt
package com.hw.hwdroid.foundation.app.annotation

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@Retention(RetentionPolicy.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class HContentViewAttachToRoot(val value: Boolean = false)


//
//@Target(ElementType.TYPE)
//@Retention(RetentionPolicy.RUNTIME)
//public @interface HContentViewAttachToRoot {
//
//    boolean value() default false;
//
//}
